.kokoro/build.sh

```diff
@@ -21,6 +21,9 @@ cd github/releasetool
 # Disable buffering, so that the logs stream through.
 export PYTHONUNBUFFERED=1
 
+# Ensure install works with requirements.txt
+python3 -m pip install --require-hashes -r requirements.txt
```

Comment:
Add installation with requirements step to ensure `requirements.txt` works in Python 3.6 environment.